src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,26 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+
+    private static class EmployeeComparator implements Comparator<Employee> {
```

Comment:
You shouldn't create a class like this here.
```suggestion
    private Comparator<Employee> employeeComparator = new Comparator<>() {
```